modules/ipi-install-bmc-addressing-for-dell-idrac.adoc

```diff
@@ -77,13 +77,13 @@ platform:
           disableCertificateVerification: True
 ----
 
-
 [NOTE]
```

Comment:
We are not supposed to stack admonitions. The preferred way to do this is to follow the guidance in the [SSG](https://redhat-documentation.github.io/supplementary-style-guide/#admonitions) and place the less important admonition in the body of the text. If it is important to have all of the content in an admonition, follow the [ISG](https://www.ibm.com/docs/en/ibm-style?topic=format-notes#standard-notes) guidance and use a bulleted list. If you follow the ISG guidance, you cannot change NOTE > NOTES, as it does not render properly.

/remove-label merge-review-in-progress
/remove-label merge-review-needed

Reply:
Correct, "NOTES" is not correct AsciiDoctor syntax [Link](https://docs.asciidoctor.org/asciidoc/latest/blocks/admonitions/). Originally, the author merged two notes into one, which was incorrect.

Reply:
The IBM guidance does not work well with our syntax and I would have to use some alternative headings that go against our admonition syntax. This would bring inconsistency into our doc set with regard to admonitions. I'm suggesting to move one note in the preceding paragraph. 

![Screenshot from 2023-06-06 10-07-00](https://github.com/openshift/openshift-docs/assets/57954076/cbb93667-4993-4724-984d-d7ee121acf8e)